nibabel/freesurfer/io.py

```diff
@@ -44,24 +45,52 @@ def _fread3_many(fobj, n):
     return (b1 << 16) + (b2 << 8) + b3
 
 
-def read_geometry(filepath):
+def _read_volume_info(fobj):
+    volume_info = OrderedDict()
+    head = np.fromfile(fobj, '>i4', 3)
+    if any(head != [2, 0, 20]):
```

Comment:
Oh, I see, this is trying to check list equality. In that case, you'd want `if not np.array_equal(head, [2, 0, 20])`, but consider also [these files](https://github.com/nipy/nibabel/issues/456#issuecomment-223835825), which are just 20.

So maybe:

    if head == 20 or np.array_equal(head, [2, 0, 20]):